com.unity.shadergraph/Editor/Data/SubGraph/SubGraphAsset.cs

```diff
@@ -66,6 +67,8 @@ class SubGraphAsset : ScriptableObject, ISerializationCallbackReceiver
 
         public DataValueEnumerable<MaterialSlot> outputs => m_SubGraphData.outputs.SelectValue();
 
+        public DataValueEnumerable<Target> unsupportedTargets => m_SubGraphData.unsupportedTargets.SelectValue();
```

Comment:
Im really not terribly happy with how limiting this restriction is (if any node in the subgraph does not support a particular target type, then the whole graph is considered to not support that type). However, I am not really sure of a better way to handle this with how the current subgraphs are implemented, since we pre compile all the data with the graph and dont know about individual nodes. Will need discussing at some point.